### ISSUE TITLE
Error! Unhandled error (execa).

### ISSUE THREAD

**Issue by ghost** [jely2002/youtube-dl-gui#460]
**Describe the bug**
Cant download anything

Screenshot because i cant copy the error:
![image](https://user-images.githubusercontent.com/28512867/212571771-e867cf59-ee5c-4bc9-a9da-4b188bde36ee.png)

**Comment by H25E** [comment]
On windows download yt-dlp.exe from here:
https://github.com/yt-dlp/yt-dlp/releases

And copy it on:
C:\Users\<USER>\AppData\Local\Programs\youtube-dl-gui\resources\app.asar.unpacked\binaries\

Or if you use the portable version, copy it on the autogenerated folder.

And voila

**Comment by H25E** [comment]
Yeah but a lot of people, like me, are going to arrive here looking for a solution without knowing what's yt-dlp, where to find it and where it goes. Now it's written for everyone.

**Comment by StefanLobbenmeier** [comment]
Feel free to use the fixed version on my fork: https://github.com/StefanLobbenmeier/youtube-dl-gui

**Comment by grape-gru** [comment]
@H25E Tried this and it does not work for me

**Comment by WPY0623** [comment]
> Feel free to use the fixed version on my fork: https://github.com/StefanLobbenmeier/youtube-dl-gui

THANK YOU SO MUCH IT'S WORKS I LOVE YOU
I LITERALLY SIGN UP THIS GITHUB ACCOUNT TO EXPRESS MY THANKFUL

**Comment by himyjan** [comment]
https://github.com/StefanLobbenmeier/youtube-dl-gui/releases

or

command line version
https://github.com/jely2002/youtube-dl-gui/issues/354#issuecomment-1127269708

**Comment by jely2002** [comment]
This issue has been fixed in v3 of the app, closing this issue as a result.
